pkg/models/model_movie.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"database/sql"
+)
+
+type Movie struct {
+	ID          int             `db:"id" json:"id"`
+	Front_Image []byte          `db:"front_image" json:"front_image"`
+	Back_Image  []byte          `db:"back_image" json:"back_image"`
```

Comment:
Needs renaming to `FrontImage`, `BackImage`.

Reply:
Renamed to FrontImage and BackImage